XCUITests/ClipBoardTests.swift

```diff
@@ -10,7 +10,7 @@ class ClipBoardTests: BaseTestCase {
     //Copy url from the browser
     func copyUrl() {
         navigator.goto(URLBarOpen)
-        app.textFields["address"].press(forDuration: 3)
+        app.textFields["address"].tap()
```

Comment:
Not sure why this had to be changed, but test was failing on iPhone 11 Pro Max simulator otherwise.

Reply:
Does menu items open after tap event?

Reply:
Apparently it does